### PR TITLE
feat: default inputs for circuit compilation

### PR DESCRIPTION
**Pull request by rpalakkal**

### CODE REVIEW

**Review by ytham — CHANGES_REQUESTED** [review]
the current [quickstart circuit](https://github.com/axiom-crypto/axiom-quickstart/blob/main/app/axiom/average.circuit.ts) doesn't specify the default inputs in the .circuit.ts file itself, unfortunately. so this might mean a relatively medium/large change in terms of what we've been messaging to users to be the correct flow in terms of our docs and example repos

**Comment by rpalakkal** [comment]
I was thinking we could merge this at the same time as all the axiom-std changes, which will require documentation/examples to change anyway

**Comment by ytham** [comment]
> I was thinking we could merge this at the same time as all the axiom-std changes, which will require documentation/examples to change anyway

got it; was there a particular reason why we want to move back to having the inputs in the file itself? 

also, so just so we're completely clear: 
- we would now require the user to have `export const defaultInputs = {...}` in their circuit file 
- remove the input.json file

any thoughts @yi-sun ?

File: circuit/js/src/cliHandler/prove.ts [hunk 1]
```diff
@@ -4,11 +4,10 @@ import { getFunctionFromTs, getProvider, readInputs, readJsonFromFile, saveJsonT
 import { existsSync } from 'fs';
 
 export const prove = async (
-    circuitPath: string,
+    compiledPath: string,
+    inputFile: string,
     options: {
```
Comment (ytham) [inline circuit/js/src/cliHandler/prove.ts:9]: need to remove `inputs?` here as well

**Review by ytham — APPROVED** [review]
LGTM w/ 1 small fix

**Pull request merged**